src/Frontend/Page/Topic.hs

```diff
@@ -3,6 +3,7 @@
 {-# LANGUAGE FlexibleInstances   #-}
 {-# LANGUAGE LambdaCase          #-}
 {-# LANGUAGE OverloadedStrings   #-}
+{-# LANGUAGE RecordWildCards     #-}
```

Comment:
I really should keep the coding policy more updated, but I don't, so technically you can do this.  (-:

But I prefer not to have this extension in the code base.  I think it makes the code harder to read and less rigorous during refactor, and the only advantage I can see is that you don't have to enter the names that you're interested in in the pattern.